### DISCUSSION TITLE
前端怎么判断有没有某个角色的权限？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#266] [02.提问]
前端怎么判断有没有某个角色的权限？  类似这种 hasAnyRole("admin") 的逻辑

**Comment by zuihou** [comment]
??? 

hasAnyRole 不就是吗

**Comment by zuihou** [comment]
看下 lamp-web permissionDirect.js 怎么实现判断的吧
